### ISSUE TITLE
[Android] FATAL EXCEPTION : Context.startForegroundService() did not then call Service.startForeground()

### ISSUE THREAD

**Issue by manuhook** [codemotionapps/react-native-dark-mode#23]
Hi,

A new error on version 0.1.1 (not present on 0.1.0)

Android 8.1.0

The app crash : 
on ad logcat :

```
10-05 20:38:45.878   553   630 E ActivityManager:  +0% 21372/
10-05 20:38:45.899 21011 21011 D AndroidRuntime: Shutting down VM
10-05 20:38:45.899 21011 21011 E AndroidRuntime: FATAL EXCEPTION: main
10-05 20:38:45.899 21011 21011 E AndroidRuntime: Process: com.xxx.mobile, PID: 21011
10-05 20:38:45.899 21011 21011 E AndroidRuntime: android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground()
10-05 20:38:45.899 21011 21011 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1768)
10-05 20:38:45.899 21011 21011 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
10-05 20:38:45.899 21011 21011 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
10-05 20:38:45.899 21011 21011 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6494)
10-05 20:38:45.899 21011 21011 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-05 20:38:45.899 21011 21011 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
10-05 20:38:45.899 21011 21011 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
10-05 20:38:45.929   553 16712 I ActivityManager: START u0 {flg=0x10000000 cmp=com.xxx.mobile/com.masteratul.exceptionhandler.DefaultErrorScreen (has extras)} from uid 10177
```

**Comment by DimitarNestorov** [comment]
0.1.2 released. Let's see what exception we get this time. If you can tell me how to reproduce the issues you're getting it would help me out with resolving them.

Related #22

**Comment by manuhook** [comment]
Thank you !
I don't see this error anymore, I will let you know if I see more exception.

**Comment by DimitarNestorov** [comment]
Hey @manuhook, you might be interested to know that I just released [0.2.0-rc.1](https://www.npmjs.com/package/react-native-dark-mode/v/0.2.0-rc.1).
This should get rid of any service related issues which other users were experiencing. You can read more: #28.
